### PR TITLE
Player Redesign

### PR DESCRIPTION
**Pull request by akrol95**
- New Player design
- View Lyrics functionality
- Change current track language functionality
- New option flyout for Android
- Queue screen rearranged and made common for both platforms
- DebounceDispatcher introduced
- ImageResourceNames introduced  (common space for cross-plarform resource naming) + converters
- Removed many unused resources files

### CODE REVIEW

File: BMM.Core/ViewModels/PlayerBaseViewModel.cs [hunk 1]
```diff
@@ -28,9 +32,13 @@ public ITrackModel CurrentTrack
             get => _currentTrack;
             private set
             {
-                SetProperty(ref _currentTrack, value);
-                RaisePropertyChanged(() => Downloaded); // Since Downloaded depends on CurrentTrack we need to raise PropertyChanged
-                RaisePropertyChanged(() => IsSeekingDisabled);
+                _currentTrackDebounceDispatcher.Run(() =>
```
Comment (kkuepper) [inline BMM.Core/ViewModels/PlayerBaseViewModel.cs:35]: can you please explain in a comment why you added the debouncer

File: BMM.Core/ViewModels/QueueViewModel.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@
 namespace BMM.Core.ViewModels
 {
     // QueueViewModel is actually not dark on Android. We should change that. But right now IDarkStyleViewModel is only used by iOS
```
Comment (kkuepper) [inline BMM.Core/ViewModels/QueueViewModel.cs:13]: that comment is no longer valid

File: BMM.UI.Android/Application/Constants/AndroidScreenSizesHeight.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+namespace BMM.UI.Droid.Application.Constants
+{
+    public class AndroidScreenSizesHeight
+    {
+        public const int HD = 1280;
+        public const int FullHD = 1280;
```
Comment (kkuepper) [inline BMM.UI.Android/Application/Constants/AndroidScreenSizesHeight.cs:6]: that seems wrong? Shouldn't FullHD be 1920?

**Comment by kkuepper** [comment]
There's some information missing. E.g. I expect to always see the album and the songnumber (HV 404).

**Comment by kkuepper** [comment]
I installed the latest alpha on my phone and it feels and looks very very good. Good job!

File: BMM.UI.Android/Application/Constants/AndroidScreenSizesHeight.cs (hunk 3 above)
Comment (akrol95) [reply BMM.UI.Android/Application/Constants/AndroidScreenSizesHeight.cs:6]: You're right, good catch.

File: BMM.Core/ViewModels/PlayerBaseViewModel.cs (hunk 1 above)
Comment (akrol95) [reply BMM.Core/ViewModels/PlayerBaseViewModel.cs:35]: When changing the song on Android, exo player sends few current track updates, where few of them is just null. 

Example seuence looks like that:
Current Song => (next button clicked) => null => Current Song => null => null => Next Song

This is changing in very short time, so adding debuncer with small slippage prevents plaver view from flickering, as finally we react for only one update.

**Comment by kkuepper** [comment]
- status bar (black on black)
- Songnumber not in subtitle if already in title

**Review by kkuepper — APPROVED** [review]

**Pull request merged**